addons/l10n_pk_edi/models/account_edi_format.py

```diff
@@ -0,0 +1,470 @@
+            }
+
+    def _check_move_configuration(self, move):
+        if self.code != "pk_einvoice_1_3":
+            return super()._check_move_configuration(move)
+        error_message = []
+        for line in move.invoice_line_ids.filtered(lambda line: line.display_type == 'product'):
+            if line.display_type == 'product':
+                if line.discount < 0:
+                    error_message.append(_("Negative discount is not allowed, set in line %s", line.name))
+                if not line.product_id.l10n_pk_edi_sale_type:
+                    error_message.append(_("Sale type is not defined for line %s", line.name))
+                if not line.product_id.l10n_pk_edi_schedule_code:
+                    error_message.append(_("Schedule code is not defined for line %s", line.name))
+                hs_code = self._l10n_pk_edi_extract_digits(line.product_id.hs_code)
+                if not hs_code:
+                    error_message.append(_("HS code is not set in product line %s", line.name))
+                elif not re.match("^[a-zA-Z0-9]{8}$", hs_code):
+                    error_message.append(_(
+                        "Invalid HS Code (%(hs_code)s) in product line %(product_line)s",
+                            hs_code=hs_code,
+                            product_line=line.product_id.name or line.name
+                        )
+                    )
+        return error_message
+
+    def _needs_web_services(self):
+        self.ensure_one()
+        return self.code == "pk_einvoice_1_3" or super()._needs_web_services()
+
+    def _l10n_pk_edi_xml_invoice_content(self, invoice):
+        return json.dumps(self._l10n_pk_edi_generate_invoice_json(invoice)).encode()
+
+    def _l10n_pk_validate_seller(self, partner):
+        self.ensure_one()
+        message = []
+        if not re.match("^.{3,100}$", partner.street or ""):
+            message.append(_("- Street required min 3 and max 100 characters"))
+        if not partner.street2 and not re.match("^.{3,100}$", partner.street2):
+            message.append(_("- Street2 should be min 3 and max 100 characters"))
+        if not re.match("^.{3,50}$", partner.state_id.name or ""):
+            message.append(_("- State required min 3 and max 50 characters"))
+        if message:
+            action = {
+                "view_mode": "form",
+                "type": "ir.actions.act_window",
+                "res_id": partner.id,
+                "res_model": "res.partner",
+                "views": [[self.env.ref("base.view_partner_form").id, "form"]],
+            }
+            error_msg = '\n'.join(map(str, message))
+            raise RedirectWarning(
+                _("Please enter required details for %s \n%s", partner.display_name, error_msg),
+                action,
+                _('Go to Configuration')
+            )
+
+    def _l10n_pk_validate_buyer(self, partner):
+        self.ensure_one()
+        message = []
```

Comment:
Here you only have one message, don't make it a list.
Initialize message to None, then if you have some, set it to the message you want